app/styles/main.css.map

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I dont know anything about anything, but should we be checking in CSS sourcemaps, or should these be auto-generated via Sass+Autoprefixer or something?